src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_quantization.cpp

```diff
@@ -604,7 +608,7 @@ void prepare_quantization::prepare_asymmetric_quantization(program &p, convoluti
         wl = wl.convert_to_weights_layout(convolution_node.typed_desc()->grouped_weights_shape);
     }
     int ofm = wl.group() * wl.ofm();
-    int ifm = in0.get_output_layout().feature();
+    int ifm = in0.is_dynamic() ? in0.get_output_layout().get_partial_shape()[1].get_max_length() : in0.get_output_layout().feature();
```

Comment:
And call here `get_length()`, because we need dimension value itself, not exactly `max_length()` 

BTW, could you please add some tests?

Reply:
`int ifm = in0.get_output_layout().get_partial_shape()[feature_idx].get_length();`
No need to have different code for static/dynamic shapes here.

Reply:
Okay I see, Applied and I'll also update some tests.